### ISSUE TITLE
Consider differentiating names of pydantic models between database and parquet-based queries

### ISSUE THREAD

**Issue by samlamont** [RTIInternational/teehr#107]
Currently the enums and pydantic classes supporting the DB and parquet-based queries are in separate modules (`models/queries.py` and `models/queries_database.py`).

Some of the classes in each module have the same name (eg., `MetricQuery`, `JoinedTimeseriesQuery`) which makes it awkward when both versions need be imported in the same module.  

For example, some functions in `queries/utils.py` are called by the database and parquet-based queries (`_select_primary_count(mq)`, `_select_bias(mq)`, etc in `get_metrics()`), however the function arguments are typed using only the parquet-based models:

```
from teehr.models.queries import (
    JoinedFilter,
    MetricQuery,
    JoinedTimeseriesQuery,
    TimeseriesFilter,
)
def _select_bias(mq: MetricQuery) -> str:
    if "bias" in mq.include_metrics or mq.include_metrics == "all":
        return """, sum(secondary_value - primary_value)/count(*) as bias"""
    return ""
```
This doesn't seem to cause problems in terms of functionality, but it might be clearer and more explicit to change the class names (`MetricQueryDB` vs. `MetricQuery` for example) so we could do:
```
def _select_bias(mq: Union[MetricQuery, MetricQueryDB]) -> str:
    if "bias" in mq.include_metrics or mq.include_metrics == "all":
        return """, sum(secondary_value - primary_value)/count(*) as bias"""
    return ""
```

@mgdenno any thoughts?

**Comment by mgdenno** [comment]
@samlamont I think I follow in general.  I agree it would be better to be explicit and use both in the typing.  I think another option, beside renaming, would be to import the modules and not the classes.  In other words, like this:

```
import teehr.models.queries as tmq
import teehr.models.queries_database as tmqd

def _select_bias(mq: Union[tmq.MetricQuery, tmqd.MetricQuery]) -> str:
    if "bias" in mq.include_metrics or mq.include_metrics == "all":
        return """, sum(secondary_value - primary_value)/count(*) as bias"""
    return ""
```
I'm not certain of the pros and cons around these...

**Comment by samlamont** [comment]
@mgdenno good point that seems cleaner to me as opposed to renaming the classes.  I can include this in the latest PR #106

**Comment by samlamont** [comment]
This was addressed in PR #106
